code/procs/command_alert.dm

```diff
@@ -9,6 +9,13 @@
 	if (sound_to_play && length(sound_to_play) > 0)
 		playsound_global(world, sound_to_play, 100)
 
+#ifndef MAP_OVERRIDE_OSHAN
+	if (alert_origin == ALERT_WEATHER)
```

Comment:
this seems kinda sus, why not do some weather signal or something

Reply:
I feel like a signal is overkill here, seeing as weather alerts are just one of two kinds of the command alerts and really only in name. Changing it would probably involve a more complicated and structured approach to redoing the alerts system, which I really don't want to do

Reply:
The original, worse solution I had was to directly compare name string. so... yea

Reply:
i've left it as is, i should mention. i don't really get what you mean by weather signal to be honest.